### ISSUE TITLE
Tracking Indonesian Translation Progress (M2)

### ISSUE THREAD

**Issue by girikuncoro** [kubernetes/website#15341]
This is tracking issue of our progress throughout the 1.16 release cycle.

## Maintainers
* @girikuncoro
* @irvifa 
* @phanama
* @wahyuoi 
* @danninov 
* @qbl

## For New Contributors
To translate a page:
1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. Take only one page at a time.
3. Fork this repo, make your translations, and submit a pull request with `language/id` label!

For more detailed instruction, consult with this [JK8s guide](https://github.com/jk8s/sig-docs-id-localization-how-tos#getting-started). Before contributing, read the [glossary](https://github.com/jk8s/sig-docs-id-localization-how-tos#glossary) and the [RSNI Glosarium from Ivan Lanin](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf) to understand how to translate various technical and Kubernetes-specific terms. (If you're still not sure about something, please don't hesitate to ask in [Slack channel](https://kubernetes.slack.com/messages/CJ1LUCUHM), we won't bite 😄)

Also, we haven't written special style guide for writing Bahasa Indonesia. However, in general we can follow the [English style guide](https://kubernetes.io/docs/contribute/style/style-guide/) with some addition to italicizing imported words.

Please commit with your translations! If you find that you can't commit any more, let the maintainers know so they can assign the page to someone else.

## For maintainers
When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

- [x] Init Kubernetes ID (@girikuncoro) #13822 

When PRs are merged, make sure to mark that page as completed!

# Work Status
(🏃 Work in Progress; 👀 In Review)

This is our second milestone (M2) that we aim to complete during the Kubernetes v1.16 release cycle. We will mostly continue on translating `concept` pages, and review the translated pages. On M1, within 2 months we made a great job on translating almost 40 pages of concepts from 16 contributors! 🎉 🎉 

## For Folks interested in Reviewing Translated Pages
Instructions are TBD once we finalize what to look when reviewing the pages.

### Architecture (`content/id/docs/concepts/architecture`)
- [x] Cloud Controller (@phanama) #15335

### Cluster Administration (`content/id/docs/concepts/cluster-administration`)
- [x] Certificates (@yofriadi) #15769
- [x] Logging (@danninov) #16071
- [x] Manage Deployment (@koneko096) #16433 
- [x] Networking (@yofriadi @wahyuoi) #15986 #19420
- [x] Proxies (@girikuncoro ) #13935

### Configuration (`content/id/docs/concepts/configuration`)
- [x] Assign Pod Node (@marchellametta) #16168
- [x] Manage Compute Resources Container (@phanama) #18468
- [x] Organize Cluster Access Kubeconfig (@wahyuoi) #17035
- [x] Overview (@sulaimantok) #17378
- [x] Pod Priority Preemption (@phanama) #17992
- [ ] ~Scheduler Perf Tuning~ (got removed in 1.16)
- [x] Secret (@irvifa) #14792

### Containers (`content/id/docs/concepts/containers`)
- [x] Images (@girikuncoro) #19784

### Extend Kubernetes (`content/id/docs/concepts/extend-kubernetes`)
- [x] Extend Cluster (@prabangkoro) #16421
- [x] Poseidon Firmament Alternate Scheduler (@prabangkoro ) #17043
- [x] Service Catalog (@wahyuoi) #17990
- [x] API Server Aggregation (@2pai) #16903
- [x] Custom Resources (@antooks) #18653
- [x] Device Plugins (@koneko096 ) #18676
- [x] Network Plugins (@prabangkoro) #17184

### Overview (`content/id/docs/concepts/overview`)
- [x] Object Management (@girikuncoro) #16854
- [x] Common Labels (@irvifa) #16706
- [x] Labels (@danninov) #19072

### Policy (`content/id/docs/concepts/policy`)
- [x]  Pod Security Policy (@phanama) #16699
- [x] Resource Quotas (@phanama) #18643

### Services Networking (`content/id/docs/concepts/services-networking`)
- [x] Add Entries to Pod etcd hosts with host aliases (@irvifa) #16009
- [x] DNS Pod Service (@irvifa) #16707

### Storage (`content/id/docs/concepts/storage`)
- [x] Dynamic Provisioning (@girikuncoro) #15583
- [x] Storage Classes (@irvifa) #15999 
- [x] Volume Snapshot Classes (@girikuncoro) #15724
- [x] Volume Snapshots (@girikuncoro) #16661
- [x] Volumes (@phanama) #16154
- [x] Volume PVC Datasource (@phanama) #17886

### Workloads (`content/id/docs/concepts/workloads`)
- [x] Daemonset (@giovanism) #18632
- [x] Deployment (@koneko096) #17815
- [x] Jobs Run to Completion (@irvifa) #16705
- [x] Replicaset (@joshuabezaleel) #16722
- [x] Replication Controller (@joshuabezaleel) #19071
- [x] StatefulSet (@irvifa) #16700
- [x] TTL After Finished (@irvifa) #16008
- [x] Init Containers (@phanama) #14927
- [x] Pod Lifecycle (@wahyuoi)#16165
- [x]  Pod (@wahyuoi) #16979

Edit: List of pages required for reviews have been removed since it confused several contributors. Will do this in separate issue.

*Note: This contribution template is inspired by Indonesian folks translating [reactjs doc](https://github.com/reactjs/id.reactjs.org/issues/1)*

Related Docs:
1. [RSNI glossary](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf)
2. [Branching Strategy](https://kubernetes.io/docs/contribute/localization/#branching-strategy)

**Comment by phanama** [comment]
I'm taking `Pod Security Policy` in `content/id/docs/concepts/policy`

**Comment by danninov** [comment]
I'm taking Persistent Volumes in content/id/docs/concepts/storage

**Comment by irvifa** [comment]
I'm taking `Storage Classes`

**Comment by yofriadi** [comment]
Hi, I will take `Certificates` in `Cluster Administration`, thank you.

**Comment by girikuncoro** [comment]
I'm taking `Volume Snapshot Classes` next in `content/id/docs/concepts/storage`

**Comment by girikuncoro** [comment]
> I'm taking Persistent Volumes in content/id/docs/concepts/storage

@danninov just realized PersistentVolumes actually has been translated, it's in the list required for reviews. I guess the review list is confusing, let me just remove it and raise another issue for review list.

**Comment by girikuncoro** [comment]
I'm taking `Volume Snapshots` next in `content/id/docs/concepts/storage`

**Comment by danninov** [comment]
> > I'm taking Persistent Volumes in content/id/docs/concepts/storage
> 
> @danninov just realized PersistentVolumes actually has been translated, it's in the list required for reviews. I guess the review list is confusing, let me just remove it and raise another issue for review list.

Ah okay. I'll take another issue then.

**Comment by danninov** [comment]
I'm taking `Logging` at `concepts/cluster-administration`

**Comment by yofriadi** [comment]
Hi all, I will take Networking in Cluster Administration, thank you.

**Comment by irvifa** [comment]
I'm taking `TTL After Finished`

**Comment by irvifa** [comment]
I'm taking `Add Entries to Pod etcd hosts with host aliases`

**Comment by irvifa** [comment]
I'm taking `StatefulSet`

**Comment by danninov** [comment]
I'm taking `Proxies` at `concepts/cluster-administration`

**Comment by danninov** [comment]
I'm taking `Labels` at `content/id/docs/concepts/overview`

**Comment by girikuncoro** [comment]
@koneko096 @marchellametta @digembok @andriiginting @yohanesmario @wahyuoi folks, your names are listed in the pages that you wanted to translate, let me know the update if you are still working on it or couldn't find time to work on it

if I don't hear from you by this weekend (1 September), I will clear them up to let other folks claim the pages, thanks for your contribution so far! 🎉

**Comment by wahyuoi** [comment]
Hi, I'm taking `Pod` in `content/id/docs/concepts/workloads`

**Comment by prabangkoro** [comment]
#16421

**Comment by prabangkoro** [comment]
#16421

**Comment by girikuncoro** [comment]
I have cleaned up dangling page claims that didn't get response from the contributors. Will be extending this M2 tracker till the end of October.

/lifecycle active

**Comment by prabangkoro** [comment]
Hi all, next I'll continue on `Extend Cluster` topic `Poseidon Firmament Alternate Scheduler` 
beforehand, I'll continue some fixes from previous PR #16421

**Comment by girikuncoro** [comment]
I'm taking `Proxies` page from cluster administration next

**Comment by irvifa** [comment]
I'm taking Jobs Run to Completion

**Comment by irvifa** [comment]
I'm taking dns-pod-service

**Comment by irvifa** [comment]
I'm taking images.

**Comment by irvifa** [comment]
I'm taking common-labels.

**Comment by koneko096** [comment]
I'm taking `Deployment` from controllers section

**Comment by joshuabezaleel** [comment]
I'm taking ReplicaSet

**Comment by 2pai** [comment]
i'm taking API Server Aggregation

**Comment by girikuncoro** [comment]
I'm taking Object Management from `content/id/docs/concepts/overview`

**Comment by wahyuoi** [comment]
I'm taking from Organize Cluster Access Kubeconfig from `content/id/docs/concepts`

**Comment by wahyuoi** [comment]
Hi, I'm taking another one, `Service Catalog` from `Extend Kubernetes`

**Comment by prabangkoro** [comment]
Hi, I'm taking `Network Plugins` from `Extend Kubernetes`

**Comment by sulaimantok** [comment]
I'am taking Organize Cluster Access Kubeconfig

**Comment by phanama** [comment]
I'm taking `Overview of Cloud Native Security`  from `content/id/docs/concepts/security/overview`

**Comment by phanama** [comment]
I'm taking `CSI Volume Cloning` from `content/id/docs/concepts/storage/volume-pvc-datasource/`

**Comment by phanama** [comment]
I'm also taking `Pod Priority and Preemption` from `content/id/docs/concepts/configuration/pod-priority-preemption/`

**Comment by phanama** [comment]
I'm taking `Managing Compute Resources for Containers` from `content/id/docs/concepts/configuration/manage-compute-resources-container/`

**Comment by phanama** [comment]
And `Resource Quotas` from `content/id/docs/concepts/policy/resource-quotas`

**Comment by joshuabezaleel** [comment]
I am taking `Replication Controller`.

**Comment by giovanism** [comment]
Can I try taking `Daemonset`?

**Comment by irvifa** [comment]
@giovanism sure, go ahead 🙂

**Comment by koneko096** [comment]
I'm taking `Device Plugins`

**Comment by antoooks** [comment]
Hi! I would like to take `Custom Resource` in `content/id/docs/concepts/extend-kubernetes`

**Comment by girikuncoro** [comment]
@joshuabezaleel @danninov @irvifa folks, let me know if you are still working on the claimed pages (the last 3 woohoo), otherwise I will reopen the pages for contribution, since we are super close in closing this M2 issue thanks to all of us 🎉 

let me know by 12 Feb 2020, otherwise I would assume reopen by default

**Comment by joshuabezaleel** [comment]
Truly sorry that I missed this. Will finish it by this weekend. @girikuncoro

**Comment by ariscahyadi** [comment]
Hi! I am working on: 
- content/id/docs/concepts/architecture/controller
- content/id/docs/concepts/containers/overview
- content/id/docs/concepts/containers/container-environment

**Comment by girikuncoro** [comment]
We have finally completed 100% of this milestone, thank you very much for everyone who has contributed and thanks for all hard work! 🎉 🎉 🎉  We will have planning meeting in the community for figuring out next milestone (M3) for SIG Docs ID, stay tuned.

Some interesting metrics that we have gathered so far (ref: https://circleci.com/gh/jk8s/sig-docs-id/tree/master):
* Total 109 pages have been localized into Bahasa Indonesia
* Total 45 contributors have contributed to kubernetes.io/id

Looking forward to continue working together! Closing this now, we'll post link to the new issue tracker once we have it.
